quickjs.c

```diff
@@ -43821,6 +43835,24 @@ static JSValue js_regexp_escape(JSContext *ctx, JSValue this_val,
     return string_buffer_end(b);
 }
 
+static JSValue js_regexp_get_leftContext(JSContext *ctx, JSValue this_val) {
+    if (ctx->regexp_last_match_start >= 0) {
+        JS_FreeValue(ctx, ctx->regexp_left_ctx);
```

Comment:
```suggestion
        JS_FreeValue(ctx, ctx->regexp_left_ctx);
        ctx->regexp_left_ctx = JS_UNDEFINED;
```
Avoids double frees, etc. Applies to the other JS_FreeValue calls below too.